immuni_exposure_reporting/helpers/validation.py

```diff
@@ -11,10 +11,13 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+import re
 import sys
 
 from immuni_common.core.exceptions import SchemaValidationException
 
+_VALID_COUNTRY_REGEX = r"^[A-Z]{2}$"
```

Comment:
IMHO, this variable should go in common and used both here and in the `CountriesListValidator` (I do not remember the exact name).